src/Project/functions/transformAndWriteDeclarationFiles.ts

```diff
@@ -0,0 +1,17 @@
+import { PathTranslator } from "@roblox-ts/path-translator";
+import transformPathsTransformer from "Project/transformers/builtin/transformPaths";
+import { assert } from "Shared/util/assert";
+import ts from "typescript";
+
+export function transformAndWriteDeclarationFiles(
+	program: ts.Program,
+	pathTranslator: PathTranslator,
+	files: Array<ts.SourceFile>,
+) {
+	const transformResult = ts.transform(files, [transformPathsTransformer(program, {})], program.getCompilerOptions());
+	const printer = ts.createPrinter({});
+	for (const sourceFile of transformResult.transformed) {
+		assert(!ts.isBundle(sourceFile));
```

Comment:
Why is this assert safe? it should have an explanation comment and/or fail message

Reply:
Should add a comment, yeah. This is because we don't support `outFile` (or really plan to)